GLOSSARY.md

```diff
@@ -0,0 +1,21 @@
+## Glossary
+
+[Early draft, please add new definitions, fix the existing ones, add links and correct organizational details!]
+
+* **AFDKO** [Adobe Font Development Kit for OpenType](http://adobe-type-tools.github.io/afdko/AFDKO-Overview.html).
+* **AHG** ad-hoc group, a group of people representing multiple stakeholders, who meet informally to discuss technical proposals (example: the AHG who got together in 2015–2016 and made OpenType Variations happen), perhaps bypassing corporate policies on formal meetings.
+* **CG** Community Group, in a font technology context probably the W3C Font and Text Community Group established 2020.
+* **FEA** OpenType Feature File (extension .fea), [afdko: OpenType™ Feature File Specification](http://adobe-type-tools.github.io/afdko/OpenTypeFeatureFileSpecification.html).
+* **FEE** Font Engineering language with Extensibility, a language for creating font feature code that can be compiled to FEA using [fontFeatures](https://github.com/simoncozens/fontFeatures/blob/master/docs/fee-format.rst).
+* **FTCG** the W3C Font and Text Community Group.
+* **IEC** International Electrotechnical Commission: alongside ISO, jointly responsible for the OFF specification.
+* **ISO** International Organization for Standardization, ultimately responsible for the OFF specification via the MPEG Working Group.
+* **MOFF** “Microsoft Open Font Format”, meaning the same as Open Font Format but emphasizes that the specification is currently controlled by Microsoft.
+* **MPEG** Motion Picture Experts Group is a Working Group established by ISO and IEC to standards for “Coding of audio, picture, multimedia and hypermedia information” including fonts.
+* **MSOT** “Microsoft OpenType”, meaning the same as OpenType but emphasizes that the specification is currently controlled by Microsoft.
+* **OFF** Open Font Format, a mirror of the current OpenType specification, unencumbered by the “OpenType” trademark.
+* **OT** OpenType, the extension of TrueType developed by Microsoft and Adobe in the late 1990s, the current industry standard specification for producing fonts, the specification maintained by Microsoft.
+* **TT** TrueType, implying the pre-OpenType font specification, the TrueType flavour of OpenType, or the quadratic curve format.
```

Comment:
The "...quadratic curve..." part looks a bit too much information, unless you also mention/reference Postscript /CFF and its cubic splines. Also, mentioning "Apple" would be good here. I would probably suggest getting rid of the former and adding the latter, or if you want to keep the former, add a 'See also... Postscript / CFF entries...'.